modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -42,3 +45,10 @@ topk(10,count by (job)({__name__=~".+"}))
 ** *If the metrics relate to a user-defined project*, review the metrics key-value pairs assigned to your workload. These are implemented through Prometheus client libraries at the application level. Try to limit the number of unbound attributes referenced in your labels.
 
 ** *If the metrics relate to a core {product-title} project*, create a Red Hat support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
+
+. Check the TSDB status in the Prometheus UI.
+.. In the *Administrator* perspective, navigate to *Networking* -> *Routes*.
+.. Select the `openshift-monitoring` project in the *Project:* list.
```

Comment:
I don't know if it also applies to labels (which I assume this is), but ISG normally says not to include punctuation in GUI elements. Could this just be `*Project*` ?